pages/store/index.vue

```diff
@@ -130,6 +146,18 @@ const shouldLoadMore = useElementVisibility(infiniteScrollDetectorElement)
 const { handleError } = useErrorHandler()
 const isMobile = useMediaQuery('(max-width: 768px)')
 
+const authorName = computed(() => getRouteQuery('author', ''))
+const publisherName = computed(() => getRouteQuery('publisher', ''))
```

Comment:
How about an explicit const names for query?
```suggestion
const queryAuthorName = computed(() => getRouteQuery('author', ''))
const queryPublisherName = computed(() => getRouteQuery('publisher', ''))
```